.github/workflows/update-aports.yml

```diff
@@ -22,9 +22,17 @@ jobs:
           # Persisted credential has higher priority than .netrc.
           # Disable it to use the correct credential.
           persist-credentials: false
+
       - name: Setup environment variables
         run: echo ${{ matrix.env }} | xargs -n1 echo >>${GITHUB_ENV}
 
+      - name: Cache distribution cache data
+        uses: actions/cache@v4
+        with:
+          path: ${{ env.ROS_DISTRO }}-cache.yaml.gz
```

Comment:
Is it OK to ignore the `ALPINE_VERSION` value in `path`, `key` and `restore-key`?

Reply:
Distribution cache is generated from the distribution.yaml and not affected by the environment.